src/clients/viacep.client.ts

```
@@ -0,0 +1,39 @@
+    }
+
+    const data: ViaCepResponse = await response.json()
+
+    if ('erro' in data) {
+      throw new Error("CEP não encontrado.")
+    }
+
+    return data
+  } catch (error) {
+    throw error
+  }
```

Biome

[error] The catch clause that only rethrows the original error is redundant.

These unnecessary catch clauses can be confusing. It is recommended to remove them.

(lint/complexity/noUselessCatch)